content/privacy-policy/privacy-policy.en-US.md

```diff
@@ -3,104 +3,185 @@ title: DeFiChain's Privacy Policy
 description: Privacy Policy
 ---
 
+<div className="text-dark-1000 scroll-mt-40 text-2xl mb-2 lg:text-[32px] lg:leading-10 lg:mb-4" id="introduction">1. Introduction</div>
```

Comment:
mb-4 is not sticking in lg :o
<img width="325" alt="image" src="https://user-images.githubusercontent.com/57183851/231336539-69facb8b-b41e-4203-830b-056a47a433fd.png">